runtime/src/tests/handle_fees.rs

```diff
@@ -47,3 +48,47 @@ fn block_author_only_receives_tips() {
         );
     });
 }
+
+#[test]
+fn block_author_receives_no_tips() {
+    initial_test_ext().execute_with(|| {
+        fn author() -> AccountId {
```

Comment:
you can drop this author it is not used.

Reply:
It is user later to make sure that author free balances is 0